xml/System/DateOnly.xml

```diff
@@ -1328,11 +1349,13 @@
         <Parameter Name="result" Type="System.DateOnly" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A span containing the characters representing a date to convert.</param>
+        <param name="format">The required format of s.</param>
+        <param name="result">When this method returns, contains the DateOnly value equivalent to the date contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s is empty string, or does not contain a date that correspond to the pattern specified in format. This parameter is passed uninitialized.</param>
+        <summary>Converts the specified span representation of a date to its DateOnly equivalent using the specified format and style.
+            The format of the string representation must match the specified format exactly. The method returns a value that indicates whether the conversion succeeded.</summary>
+        <returns>
+          <see langword="true" /> if s was converted successfully; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="s" /> was converted successfully; otherwise, <see langword="false" />.</returns>
```